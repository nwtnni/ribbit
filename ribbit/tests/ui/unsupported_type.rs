#[ribbit::pack(size = 24)]
struct Bad {
    a: (u8, u16),
}

fn main() {}
