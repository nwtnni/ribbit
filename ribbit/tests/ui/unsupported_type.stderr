error: Only type paths are supported
 --> tests/ui/unsupported_type.rs:3:8
  |
3 |     a: (u8, u16),
  |        ^^^^^^^^^
